src/app/directives/highlight.directive.spec.ts

```diff
@@ -33,7 +33,7 @@ describe('HighlightDirective', () => {
     ngMocks.trigger(el, 'mouseover');
     fixture.detectChanges();
     //console.log(ngMocks.formatHtml(fixture.debugElement))
-    expect(ngMocks.formatHtml(fixture.debugElement)).toContain('box_shadow');
+    expect(ngMocks.formatHtml(fixture)).toContain('box_shadow');
```

Comment:
`ngMocks.formatHTML` accepts fixtures and debugElements, or an array of them.

Reply:
thats great! Searching through DOM to find a piece of text or a sub string is a common criteria. If fixture itself can do that, that's less work for us